airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/migrate_to_logging_logger/pipeline.py

```diff
@@ -0,0 +1,110 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+from __future__ import annotations
+
+import json
+import os
+import shutil
+import tempfile
+from dataclasses import dataclass
+from pathlib import Path
+from typing import TYPE_CHECKING, Any, List
+
+from connector_ops.utils import ConnectorLanguage  # type: ignore
+from pipelines import main_logger
+from pipelines.airbyte_ci.connectors.consts import CONNECTOR_TEST_STEP_ID
+from pipelines.airbyte_ci.connectors.context import ConnectorContext, PipelineContext
+from pipelines.airbyte_ci.connectors.reports import Report
+from pipelines.consts import LOCAL_BUILD_PLATFORM
+from pipelines.helpers.connectors.command import run_connector_steps
+from pipelines.helpers.connectors.format import format_prettier
+from pipelines.helpers.connectors.yaml import read_yaml, write_yaml
+from pipelines.helpers.execution.run_steps import STEP_TREE, StepToRun
+from pipelines.models.steps import Step, StepResult, StepStatus
+
+if TYPE_CHECKING:
+    from anyio import Semaphore
+
+
+class CheckIsInlineCandidate(Step):
+    """Check if the connector is a candidate to get replace the AirbyteLogger.
+    Candidate conditions:
+    - The connector is a Python connector.
+    """
+
+    context: ConnectorContext
+
+    title = "Check if the connector is a candidate for re."
+
+    def __init__(self, context: PipelineContext) -> None:
+        super().__init__(context)
+
+    async def _run(self) -> StepResult:
+        connector = self.context.connector
+        if connector.language not in [ConnectorLanguage.PYTHON, ConnectorLanguage.LOW_CODE]:
+            return StepResult(
+                step=self,
+                status=StepStatus.SKIPPED,
+                stderr="The connector is not a Python connector.",
+            )
+
+        return StepResult(
+            step=self,
+            status=StepStatus.SUCCESS,
+        )
+
+
+class InlineSchemas(Step):
```

Comment:
Rename the step pretty please 🙃 
```suggestion
class MigrateAirbyteLogger(Step):
```